.github/workflows/build-epiniod.yml

```diff
@@ -0,0 +1,36 @@
+name: Build epiniod image
+
+on:
+  release:
+    types: [published]
```

Comment:
Maybe we can also add the workflow dispatch trigger? This will allow us to try out the pipeline manually.

Reply:
added that